src/main/java/com/github/ozsie/CheckMojo.kt

```diff
@@ -12,14 +12,18 @@ import java.nio.file.Paths
 @Suppress("unused")
 @Mojo(name = "check",
         defaultPhase = LifecyclePhase.VERIFY,
+        threadSafe = true,
         requiresDependencyCollection = ResolutionScope.TEST)
 class CheckMojo : DetektMojo() {
     override fun execute() {
         getCliSting().forEach {
             log.debug("Applying $it")
         }
-        val cliArgs = parseArguments<CliArgs>(getCliSting().log().toTypedArray())
+        val cliArgs = parseArguments<CliArgs>(getCliSting().log().toTypedArray(), System.out, System.err)
         skip = !Files.isDirectory(Paths.get(input))
-        if (!skip) return Runner(cliArgs).execute() else log.info("Input directory '$input' not found, skipping module")
+        if (!skip)
```

Comment:
this line was too long after I updated it
and frankly - it wasn't very readable as one-liner